src/bind.js

```diff
@@ -17,6 +17,11 @@ var _curry2 = require('./internal/_curry2');
  * @param {Object} thisObj The context to bind `fn` to
  * @return {Function} A function that will execute in the context of `thisObj`.
  * @see R.partial
+ * @example
+ *
+ *      var log = R.bind(console.log, console);
+ *      R.pipe(R.assoc('a', 2), R.tap(log), R.assoc('a', 3))({a: 1}); //=> {a: 3}
+ *      //-> {a: 2}
```

Comment:
I might write something like

```js
// logs {a: 2}
```

instead.  The difference between `//=>` and `//->` is fairly subtle and quite difficult to notice.